app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script_source.rb

```diff
@@ -14,4 +14,18 @@ def provider_collection(manager)
   def provider_object(connection = nil)
     (connection || connection_source.connect).api.projects.find(manager_ref)
   end
+
+  REFRESH_ON_TOWER_SLEEP = 1.second
+  def refresh_on_tower
```

Comment:
can we use `refresh_in_provider`, to share the pattern with the `create/update/delete`?